lib/upload.js

```diff
@@ -79,7 +80,13 @@ module.exports = {
                         'directory does not contain an index.html\n' +
                         'Make sure you\'re deploying the right public directory: ' +
                         chalk.bold(path.resolve(publicDir)));
+      } else if (fileCount > 500 || tarballSize > 50*1024*1024) {
+        console.log(chalk.yellow('Public Directory Warning') + ' - Uploading ' +
```

Comment:
We use `WARNING` above, and `Warning` here - we should try and be consistent on case. I think lower-case is probably enough - depending on president considering it's already yellow